parsec/_parsec_pyi/invite.pyi

```diff
@@ -70,7 +70,7 @@ class InviteUserData:
     @classmethod
     def dump_and_encrypt(cls, key: SecretKey) -> bytes: ...
     @classmethod
-    def decrypt_and_load(cls, encrypted: bytes, key: SecretKey) -> bytes: ...
+    def decrypt_and_load(cls, encrypted: bytes, key: SecretKey) -> InviteUserData: ...
```

Comment:
`__init__` and `getter` differs:
`requested_device_label | request_human_handle`
`requested_human_handle | request_device_label`

Reply:
Yes, the other way around was better

Reply:
But you comment is really missplaced :/